_includes/os-sidebar.html

```diff
@@ -103,17 +103,6 @@
                     </ul>
                 </li>
                 <li><a href="{{site.baseurl}}/os/amazon-ecs/">Amazon ECS</a></li>
-                <li class="panel">
```

Comment:
Should we remove it or just comment it out? Would we want to add in the same kind of information at some point?